macaroonbakery/bakery.py

```diff
@@ -17,13 +17,19 @@
 DEFAULT_PROTOCOL_VERSION = {'Bakery-Protocol-Version': '1'}
 MAX_DISCHARGE_RETRIES = 3
 
+BAKERY_V0 = 0
+BAKERY_V1 = 1
+BAKERY_V2 = 2
+BAKERY_V3 = 3
+LATEST_BAKERY_VERSION = BAKERY_V1
+NONCE_LEN = 24
+
 
 class DischargeException(Exception):
     """A discharge error occurred."""
 
 
-def discharge_all(macaroon, visit_page=utils.visit_page_with_browser,
-                  jar=requests.cookies.RequestsCookieJar(), key=None):
+def discharge_all(macaroon, visit_page=None, jar=None, key=None):
     ''' Gathers discharge macaroons for all the third party
```

Comment:
Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description. See https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings